src/components/YieldPools/styleds.tsx

```diff
@@ -593,6 +593,8 @@ export const ActionButton = styled(ButtonLight)<{ backgroundColor?: string }>`
   background-color: ${({ theme, backgroundColor }) => backgroundColor || theme.primary + '33'};
   width: 28px;
   height: 28px;
+  min-width: 28px;
```

Comment:
why do we need `min-width 28px` when we already have `width 28px`.

Reply:
removed